src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -84,7 +84,7 @@ import { Action, Getter } from 'pinia-class'
 import { AddCommentDialog, DownloadErrorDialog, FileCorrectionDialog, LoadCorrectionDialog }
   from '@/components/dialogs'
 import { CorrectionTypes } from '@/enums'
-import { ActionBindingIF, ApiFilingIF, CorrectionFilingIF } from '@/interfaces'
```

Comment:
As this IF is no longer used (and the other one in that file is not used), please delete it.

Reply:
Which IF are you referring to?

Reply:
The one you stopped using in this ticket.

Reply:
Please also remove the reference to this file from the interfaces/index.ts file. It's causing a failure of the build and unit tests checks.

Before committing, you should do these to check for issues:
```
npm run lint
npm run test:unit
```